### PR TITLE
Changed to use keycloak-admin-client

### PR DESCRIPTION
**Pull request by edewit**
Now uses the `@keycloak/keycloak-admin-client` instead

### CODE REVIEW

File: test/utils/realm.js [hunk 1]
```diff
@@ -33,8 +34,10 @@ var kca = keycloakAdminClient(settings);
  */
 function createRealm (realmName) {
   var name = realmName || 'test-realm';
-  return kca.then((client) => {
-    return client.realms.create(parse(realmTemplate, name));
+  return kca.auth(settings).then(() => {
+    return kca.realms.create(parse(realmTemplate, name)).then(() => {
+      return kca.realms.findOne({ realm: realmName });
+    });
```
Comment (jonkoops) [inline test/utils/realm.js:40]: I believe that sometimes the realm name can be sanitized by the API when created, so perhaps we can use the `realmName` that is returned from the endpoint?
```suggestion
    return kca.realms.create(parse(realmTemplate, name)).then(({ realmName: realm }) => {
      return kca.realms.findOne({ realm });
    });
```

That said, does the `create()` call not return the entire `RealmRepresentation`? If so we can simply omit the `findOne()` call.

File: package.json [hunk 2]
```diff
@@ -38,18 +38,18 @@
     "jwk-to-pem": "^2.0.0"
```
Comment (jonkoops) [inline package.json:38]: Can we also check in the package lock file to keep the builds reproducible?

**Review by jonkoops — APPROVED** [review]
Couple of small comments, but nothing that I would consider blocking.

**Review by abstractj — APPROVED** [review]
LGTM thank you, @edewit 

Waiting for @jonkoops approval before merging it.

File: test/utils/realm.js (hunk 1 above)
Comment (jonkoops) [reply test/utils/realm.js:40]: Since we're only using this for tests let's keep things as they are.

**Review by jonkoops — APPROVED** [review]

**Pull request merged**